README.md

```diff
@@ -82,3 +82,16 @@ After cloning this repository, these are the 2 steps you should follow:
 2. Run `./run.sh`: this will run Jenkins with the Zephyr Scale plugin installed
 3. Browse to http://localhost:8080/jenkins to access Jenkins
 
+## Trigger pipeline manually for forked pull requests
+
+Eventually, CircleCi doesn't trigger the pipeline from pull requests of forked repositories. 
```

Comment:
@chicofaccin  thanks for adding this comment, but since this is a public repository , do you think we should have this comment in here ?

Reply:
@johnathafelix mentioned yesterday that the build wasn't triggered probably because he enabled the "build from forks" option in CircleCI just after the PR was created. It might just work for the next one, no?

Reply:
I forked the project and create a PR:
<img width="996" alt="Screenshot 2021-11-17 at 11 47 54" src="https://user-images.githubusercontent.com/1117150/142195108-7348349b-7a13-47cc-8ff7-ddc950f3a16c.png">
And It's working:

<img width="1389" alt="Screenshot 2021-11-17 at 11 48 55" src="https://user-images.githubusercontent.com/1117150/142195240-275d44ba-c603-43e7-9d54-59b4f047cd0e.png">

Reply:
That's great, so we can decline this PR I assume?

Reply:
Yes, we can!